examples/predicted_molecular_consequences/1:230710048:rs699.json

```
@@ -0,0 +1,697 @@
+{
+  "data": {
+    "variant": {
+      "name": "rs699",
+      "allele_type": {
+        "accession_id": "SNV",
+        "value": "SNV",
+        "url": "http://sequenceontology.org/browser/current_release/term/SO:0001483"
+      },
+      "alleles": [
+        {
+          "name": "1:230710048:A:G",
+          "predicted_molecular_consequences": [
+            {
+              "allele_name": "G",
+              "stable_id": "ENST00000366667",
+              "feature_type": {
+                "value": "Transcript"
+              },
+              "consequences": [
+                {
+                  "value": "missense_variant"
+                }
+              ],
+              "gene_symbol": "AGT",
+              "gene_stable_id": "ENSG00000135744",
+              "transcript_biotype": "protein_coding",
+              "protein_stable_id": "protein_id_placeholder",
+              "prediction_results": [
+                {
+                  "score": 1,
+                  "result": null,
+                  "classification": {
+                    "label": "tolerated",
+                    "definition": ""
+                  },
+                  "analysis_method": {
+                    "tool": "SIFT"
+                  }
+                },
+                {
+                  "score": 0,
+                  "result": null,
+                  "classification": {
+                    "label": "benign",
+                    "definition": ""
+                  },
+                  "analysis_method": {
+                    "tool": "PolyPhen"
+                  }
+                }
+              ],
+              "cdna_location": {
+                "start": 843,
+                "end": 843,
+                "length": 1,
+                "ref_sequence": "A",
+                "alt_sequence": "G"
```

```suggestion
                "ref_sequence": "T",
                "alt_sequence": "C"
```

and others of these variant would be same.